cip/1.accepted/CIP2015-06-24-call-procedures.adoc

```diff
@@ -190,31 +192,32 @@ RETURN *
 
 ===== Calling with implicitly passed arguments (parameters) and with `YIELD *`
 
-Both simplifications may be used in a single standalone procedure call, leading to a very concise syntax for just executing a single procedure call:
+Both simplifications may be used in a single standalone procedure call, leading to a concise syntax for just executing a single procedure call:
 
-.Simplified standalone procedure call
+.Standalone procedure call with `YIELD *`
 [source, cypher]
 ----
 CALL myProc YIELD *
 ----
 
-==== Standalone call to a procedure that does not return result fields
+===== Calling with implicitly passed arguments (parameters) and omitting `YIELD`
 
-Standalone calls without `YIELD` are only supported for procedures that do not return result fields:
+As a further simplification, the `YIELD` subcluase can be omitted alltogether from a standalone procedure call, leading to an even more concise syntax:
 
-.Standalone call to a procedure that does not return result fields
+.Standalone procedure call with omitted `YIELD` and implicitly passed arguments
 [source, cypher]
 ----
-CALL myProc($arg)
+CALL myProc
 ----
 
-Omitting all result fields when calling a procedure may still be achieved using an in-statement call:
+==== Standalone call with omitted `YIELD` clause
```

Comment:
This and the section on line 207 feels a bit repetitive, especially directly after each other

Reply:
This section should be part of standalone call section? Not on the same level?